### PR TITLE
added categories' table and model

### PR DESCRIPTION
**Pull request by DwinaTech**

### CODE REVIEW

File: backend/.eslintrc [hunk 1]
```diff
@@ -16,6 +16,7 @@
       "object-shorthand": 0,
       "padded-blocks": 0,
       "arrow-parens": 0,
-      "babel/arrow-parens": 2
+      "babel/arrow-parens": 2,
+      "linebreak-style": "off"
```
Comment (kkarimi) [inline backend/.eslintrc]: Is this necessary? if so maybe we just turn it on for windows: `linebreak-style: ["error", "windows"]`

File: backend/.eslintrc [hunk 2]
```diff
@@ -16,6 +16,7 @@
       "object-shorthand": 0,
       "padded-blocks": 0,
       "arrow-parens": 0,
-      "babel/arrow-parens": 2
+      "babel/arrow-parens": 2,
```
Comment (kkarimi) [inline backend/.eslintrc]: Is this one real thing? doesn't the line above cover it?

File: backend/src/controllers/dml.js [hunk 3]
```diff
@@ -5,28 +5,28 @@ import Organisation from '../model/Organisation';
...
-      const organisation = await Organisation
-        .query(trx)
-        .insertGraph(query);
-      await trx.commit();
-      console.log('Data is inserted !')
-    } catch (err) {
-      await trx.rollback();
-      console.log('Something went wrong. Data is not inserted', err);
-    }
+  let trx
```
Comment (kkarimi) [inline backend/src/controllers/dml.js]: I'd remove this file because @sentayhuJo has been working on it

File: backend/.eslintrc (hunk 2 above)
Comment (DwinaTech) [reply backend/.eslintrc]: it is exist just one time

**Pull request merged**